system/src/ble_control_request_channel.cpp

```diff
@@ -603,12 +604,14 @@ void BleControlRequestChannel::run() {
         resetChannel();
```

Comment:
This method should probably check for initialization state and return early if not initialized.